src/sign.proj

```diff
@@ -14,10 +15,25 @@
       <SignBuildArgs Include="/p:OfficialBuildId=$(OfficialBuildId)" />
       <SignBuildArgs Include="/p:SignType=$(SignType)" />
     </ItemGroup>
-    
+
+    <PropertyGroup>
+      <VSWhereVersion Condition="'$(VSWhereVersion)' == ''">2.5.2</VSWhereVersion>
+    </PropertyGroup>
+
+    <!-- We only need this if we are going to use the executable version. -->
+    <Exec Command='"$(NuGetPackageRoot)\vswhere\$(VSWhereVersion)\tools\vswhere.exe" -latest -prerelease -property installationPath -requires Microsoft.Component.MSBuild'
```

Comment:
Should we do the same for resgen.exe in `src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj`?

Reply:
No. I don't think it is worth doing that in the ResourceManager project, especially given we rarely changes the resources.

Reply:
Ah I just saw that you changed the resources target to be invoked on demand. Got it.